Tests/Integration/Transaction/TransactionIdempotenceTests.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  Copyright (c) 2020-2021 MobileCoin. All rights reserved.
+//
+// swiftlint:disable empty_xctest_method
+
+@testable import MobileCoin
+import XCTest
+
+#if swift(>=5.5)
```

Comment:
fix needed before approval

Reply:
Fixed in P R #188, reverted to old nested closure format w/out async.